### PR TITLE
Milestone 3: Contact form

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this PR regarding the mobile-first design of the portfolio project, we have added:

1. A contact form in the footer containing input fields of full name, email address, textarea, and a button.
2. A bottom navigation section containing social media icons list.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -542,34 +542,125 @@ nav {
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-input[type="submit"]:disabled {
+button:hover {
+  cursor: pointer;
+  color: brown;
+}
```
Comment (mckent05) [inline style.css:656]: - Please implement the button interactions for your form button (I'd recommend you do these for all other buttons also) 🙏 . Please remember replicating exact Figma templates gives confidence to recruiters in your ability to replicate business requirements as regards design 😄 

Figma template (button interactions to be implemented, pay attention to the bg color, text color, and border colors):
button interact

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi guys, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are passing ✔️ ✔️ 
- Descriptive PR summary and PR title 👏 👏 
- Your portfolio is gradually coming together and it's looking great 💯 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AbeeraTahir) [reply style.css]: Hi @mckent05,

Yes I have now implemented all of these designs according to Figma but I think their behavior is hidden due to button-enabled implementation

**Review by sajid-munawar — APPROVED** [review]
### Hi @AbeeraTahir , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-munawar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**